src/site/content/en/blog/css-size-adjust/index.md

```diff
@@ -0,0 +1,395 @@
+---
+title: CSS `size-adjust` for `@font-face`
+subhead: As a web font loads, you can now adjust its scale, to normalize the document font sizes and prevent layout shift when switching between fonts
+authors:
+  - adamargyle
+description: As a web font loads, you can now adjust its scale to normalize the document font sizes and prevent layout shift when switching between fonts
+hero: image/vS06HQ1YTsbMKSFTIPl2iogUQP73/fL3dWSY7YBfagFP0ikMW.jpg
+alt: Well used letters from a letterpress, set into rows
+tags:
+  - blog
+  - css
+date: 2021-06-04
+updated: 2021-06-04
+---
+
+Consider the following
+[demo](https://codepen.io/argyleink/pen/8ace843980e2fd24bf15c8bd1bd072e4?editors=1100)
+where the `font-size` is a consistent `64px`, and the only difference between each of these headers is the `font-family`. The examples on the left have not been adjusted and
+have an inconsistent final size. The examples on the right use `size-adjust` to
+ensure `64px` is the consistent final size.
+
+<figure class="w-figure w-figure--fullbleed">
+  {% Video
+    src="video/vS06HQ1YTsbMKSFTIPl2iogUQP73/6xhatUhszn7M6pDFcVgV.mp4",
+    autoplay="true",
+    loop="true",
+    muted="true"
+  %}
+  <figcaption class="w-figure">
+   In this example Chrome DevTools CSS grid layout debug tools are used to show heights.
+  </figcaption>
+</figure>
+
+This post explores a [new CSS font-face
+descriptor](https://drafts.csswg.org/css-fonts-5/#size-adjust-desc) called
+`size-adjust`, available in Firefox 89 (behind a flag) and Chromium 92 (see
+[caniuse](https://caniuse.com/mdn-css_at-rules_font-face_size-adjust) for latest
+support). It also demonstrates a few ways to correct and normalize font sizes
+for smoother user experience, consistent design systems and more predictable
+page layout. For example, when optimizing initial font rendering to prevent
+[cumulative layout shift](/cls/) (CLS). 
+
+Here's an interactive [demo](https://codepen.io/argyleink/pen/rNyMjxR) of the
+problem space. Try changing the typeface with the dropdown and observe: 
+1. The wildheight differences in the results.
+1. Visual jarring content shifts. 
+1. Moving interactive target areas (the dropdown jumps around!).
+
+{% Codepen {
+  user: 'web-dot-dev',
+  id: 'WNpzRKd'
+} %}
+
+{% Aside 'key-term' %} 
+Font family vs Typeface: A **typeface** is referred to by
+its family name plus its font face. `Helvetica Bold` is referring to the
+specific bold typeface while `Helvetica` is generically referring to the entire
+family of 8+. With CSS `@font-face` you'll be dealing with typefaces, even
+though to use them you need to write `font-family`. 
```

Comment:
Suggest removing this "even though", or rephrasing to make clear that it makes sense because there are multiple fields in an `@font-face` that *together* form the parameters indicating which typeface to use.